lib/rollout.rb

```diff
@@ -159,6 +166,33 @@ def define_group(group, &block)
     @groups[group.to_sym] = block
   end
 
+  def define_id_collection(name, ids = [])
+    @storage.set("collection:#{name}", ids.join(","))
+    @storage.set(collection_key, (collections.keys | [name.to_sym]).join(","))
```

Comment:
@gschammah any opinion in bitwise or for array operations? I've never seen it in our repos, but maybe its better than ```ary.concat(other_ary).uniq```?